models/examples/netbox/netbox.yml

```diff
@@ -831,7 +831,7 @@ nodes:
       - name__value
       - vlan_id__value
     uniqueness_constraints:
-      - ["location", "vlan_group", "vlan_id__value"]
+      - ["name__value", "vlan_id__value"]
```

Comment:
Wouldn't this become an issue if you have VLAN ["server", 23] in 2 locations?

Reply:
Indeed, i should maybe remove the uniqueness constraint completely on this one for now